collector/collector/config.py

```diff
@@ -35,6 +35,9 @@ class Config(object):
     # Architecture.
     TELEMETRY_ID = "6907c830-eed9-4ce9-81ae-76daf8d88f0f"
 
+    # Upload files path
+    ATTACHMENT_QUARANTINE_FOLDER = '/tmp/clrtelemetry.attachments/'
```

Comment:
Is `/tmp` is good place to store the attachments, or should they be kept in a directory that the app owns? For example, `/var/www/telemetry/collector/attachments`?

Reply:
I'll use ```/var/www/telemetry/collector/attachments``` for production and ```/tmp/attachments``` for testing.

Reply:
Sounds good to me